config/bash_profile

```diff
@@ -39,9 +39,9 @@ fi
 
 # Set the WP_TESTS_DIR path directory so that we can use phpunit inside
 # plugins almost immediately.
-export WP_TESTS_DIR=/srv/www/wordpress-develop/public_html/tests/phpunit/
+export WP_TESTS_DIR=/srv/www/wordpress-trunk/public_html/tests/phpunit/
```

Comment:
we should check if this folder exists, then fall back to `wordpress-develop` if that exists, the same with `WP_CORE_DIR`

Reply:
That's a good idea. I will update the PR

Reply:
The old directory is also used here.
https://github.com/Varying-Vagrant-Vagrants/VVV/blob/f95ea2082e44621e8abaf0b402f5b94ec5dff269/config/homebin/makepot#L4

Should I also update that (with a conditional for the new directory)? I'm not sure if it's called anywhere.

This is also using the old directory.
https://github.com/Varying-Vagrant-Vagrants/dashboard/blob/21858cfb44785bb0800a076e997f845986f63cd3/php/blocks/main/sites.php#L23

But that, I think, should be handled separately.

Reply:
I think it should, but lets put that in another PR. As for the dashboard repo, I think that's fine to keep as is, it's mostly adding a description for much older VVV users that didn't use `description` tags

Reply:
Ok, thanks

Reply:
I've updated the PR to check the directories before setting the variables.

Reply:
Thanks :)